### ISSUE TITLE
[Proposal] DataFusion monthly online meetup

### ISSUE THREAD

**Issue by edmondop** [apache/datafusion#10845]
### Is your feature request related to a problem or challenge?

DataFusion is such an exciting project, with so many great contributors and so many users that are building useful systems on top of it. It would be great get together, entirely online, once per month for one hour.

### Describe the solution you'd like

We can meet through the Discord meeting feature, maybe even find a way to also have breakout rooms, where subgroups might meet to discuss a specific issue, a specific idea, a specific theme. 

### Describe alternatives you've considered

Keeping the interactions fully asynchronous, using GitHub issues, discord or slack. 

### Additional context

I have recently took part to the Apache DataFusion Comet bi-weekly community meeting and I loved it.

**Comment by mustafasrepo** [comment]
I think, this would be very nice.

**Comment by alamb** [comment]
I agree -- this would be very nice. We had something like this a long time ago, but it ran out of steam due to lack of an organizer (one challenge is that due to timezones it is almost impossible to find a time that works well for everyone). 

I suggest you simply schedule something @edmondop  and I will certainly do my best to attend . Maybe you can provide details here in this ticket (or make a PR to update https://datafusion.apache.org/contributor-guide/communication.html if you plan to do it regularly)

Thank you
